src/codeu/chat/common/ConversationHeader.java

```diff
@@ -75,6 +77,8 @@ public ConversationHeader read(InputStream in) throws IOException {
 	// a hashmap with user id as key and their usertype as value
 	private HashMap<Uuid, UserType> userAccessRoles = new HashMap<Uuid, UserType>();
 
+	public LinkedList<String> bots = new LinkedList<String>();
```

Comment:
Does this have to be public? Did you choose LinkedList (as opposed to other data structures) for a specific reason? Comments comments comments.